roles/network_plugin/calico/tasks/main.yml

```diff
@@ -94,7 +94,7 @@
   shell: >
     echo '{
     "kind": "ipPool",
-    "spec": {"disabled": false, "ipip": {"enabled": {{ cloud_provider is defined or ipip }}},
+    "spec": {"disabled": false, "ipip": {"enabled": {{ calico_ipip is defined | ternary(calico_ipip, cloud_provider is defined) }}},
```

Comment:
Use default filter instead of ternary:

`{"enabled": {{ calico_ipip | default(cloud_provider is defined) }}},`